tests/testfiles/CommonMark_0.30/list_items/044.test

```diff
@@ -1,30 +1,28 @@
 %   ---RESULT--- "example": 296,
 %   
 %   <ol start="10">
-%   <li>foo
+%   <li><em>foo</em>
 %   <ul>
-%   <li>bar</li>
+%   <li><em>bar</em></li>
 %   </ul>
 %   </li>
 %   </ol>
 %   
 %   ---\RESULT---
 
-\def\markdownOptionFancyLists{true}
```

Comment:
[The `fancy_lists` syntax extension][1] does way more than just support the `)` character, which is [what CommonMark requires][2]. Therefore, we will want to update the base grammar of Markdown rather than enable the `fancy_lists` syntax extension.

 [1]: https://pandoc.org/MANUAL.html#extension-fancy_lists
 [2]: https://spec.commonmark.org/0.30/#ordered-list-marker